examples/nlp/scripts/process_wiki_zh.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# =============================================================================
+# Copyright 2019 NVIDIA Corporation. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+import os
+import json
+from concurrent.futures import ProcessPoolExecutor
+from functools import partial
+import re
+
+
+def create_vocab(lines,
+                 vocab_file,
+                 min_frequency=3,
+                 special_symbols=("[PAD]", "[SEP]", "[CLS]",
+                                  "[MASK]", "[UNK]")):
+    """Create vocabulary from lines"""
+    # Count word occurency
+    vocab = {}
+    for line in lines:
+        if line.strip():
+            for w in line.strip().split():
+                if w in vocab:
+                    vocab[w] += 1
+                else:
+                    vocab[w] = 1
+
+    # Remove rare words
+    for w in vocab.keys():
+        if vocab[w] < min_frequency:
+            del vocab[w]
+            print("Remove rare word {} from vocab".format(w))
+
+    # Include special symbols and write to file
+    vocab = list(special_symbols) + sorted(vocab.keys())
```

Comment:
special_symbols can be passed in as a list so you don't have to convert it later

Reply:
Thanks! Revised this.